### ISSUE TITLE
Accordion Example: When a section header cannot be collapsed, it should have aria-disabled

### ISSUE THREAD

**Issue by mcking65** [w3c/aria-practices#455]
When the [accordion example](http://w3c.github.io/aria-practices/examples/accordion/accordion.html) is configured so that one section must be expanded, the the header for the expanded section should have aria-disabled set to true.

Note: attention @gerardkcohen

**Comment by mcking65** [comment]
Fixed with commit fa70af2; closing.
